lib/socketpair.h

```diff
@@ -34,4 +34,11 @@ int Curl_socketpair(int domain, int type, int protocol,
 #define Curl_socketpair(a,b,c,d) socketpair(a,b,c,d)
 #endif
 
+#ifdef MPTCP
+#ifndef IPPROTO_MPTCP
+#define IPPROTO_MPTCP 262
+#endif
+#endif
+
+
```

Comment:
Why duplicate this snippet?

Reply:
This is mistake, apologies. Windows does not support MPTCP and nothing needs to be changed in this file